src/main/resources/db/migration/V1.13__Update_deckPrice_column_name.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE deck_price CHANGE COLUMN deck_price price INT NOT NULL default 0;
```

Comment:
Just a double check: don't you need the same for COURSE_PRICE?

Reply:
No, because in the course_price table column named price.